settings.gradle

```diff
@@ -2,12 +2,12 @@
   This is the API (major + minor) version of Minecraft we are
   building against.
 */
-gradle.ext.apiVersion = '1.16'
+gradle.ext.apiVersion = '1.17'
 
 /*
   The Spigot version we are building MockBukkit against.
 */
-gradle.ext.apiVersionFull = '1.16.5-R0.1-SNAPSHOT'
+gradle.ext.apiVersionFull = '1.17.1-R0.1-SNAPSHOT'
 
 /*
   MockBukkit uses semantic-versioning (https://semver.org/).
```

Comment:
Hmmm, should we bump the MockBukkit version here too?

Reply:
Ya, I've already told @seeseemelk that I'm working rn and this adventure here will be continued when I have more free time

Reply:
I am aware but I don't think that answers my question.
I am not 100% sure if we need to update the version since `MockBukkit-v1.17` will be a new artifact anyway.
Then again, things were updated, so... I would argue either works but we should definitely think about it.

Reply:
Indeed, as it's a new artifact we wouldn't have any conflicts if we keep the current version number. But perhaps it's a good idea to bump it just in case someone thinks `MockBukkit-v1.16:1.5.1` and `MockBukkit-v1.17:1.5.1` are for some reason the same?

Reply:
So how we should move on now? :D

Reply:
Just bump the version I would say.
Other than that, the redundant function from above's comment needs to get sorted out. Then it seems good to go for me.
Any reason why you still kept it as a draft?

Reply:
I wanted to add more tests.

Reply:
I can't see how any unit test would have to do with the update to 1.17 explicitly.
So I would 100% favour you making a seperate PR for that.
Small, compact PRs are also easier for us to review and therefore get approved and merged faster ;)